spec/menubar/utils_spec.lua

```diff
@@ -65,6 +65,65 @@ describe("menubar.utils lookup_icon_uncached", function()
         assert_found_in_path('awesome',  '/.icons/awesome/64x64/apps/awesome.png')
         assert_found_in_path('awesome2', '/.icons/awesome/scalable/apps/awesome2.png')
     end)
+
+-- Check for no argument :: if not icon_file or icon_file == "" then
+    it('no icon specified', function()
+        assert.is_false(utils.lookup_icon_uncached())
+        assert.is_false(utils.lookup_icon_uncached(nil)) -- Same as above?
+        assert.is_false(utils.lookup_icon_uncached(false))
+        assert.is_false(utils.lookup_icon_uncached(''))
+    end)
+
+-- Check for icon not in search path :: if icon_file:sub(1, 1) == '/' and supported_icon_formats[icon_file_ext] then
+    it('finds icons even those not in the search paths when full path specified', function()
+
+        -- Shimmed icon base directories contain the following icons:
+        --
+        --     usr/share/icon5.png
+        --     usr/share/icon6.xpm
+        --     usr/share/icon7.svg
+
+        assert_found_in_path('/usr/share/icon5.png', '/usr/share/icon5.png')
+        assert_found_in_path('/usr/share/icon6.xpm', '/usr/share/icon6.xpm')
+        assert_found_in_path('/usr/share/icon7.svg', '/usr/share/icon7.svg')
+
+        assert.is_same(nil, utils.lookup_icon_uncached('/.png')) -- supported file does not exist in location
+        assert.is_same(nil, utils.lookup_icon_uncached('/blah/icon6.png')) -- supported file does not exist in location
+    end)
+
+-- Check icon with specified extension matching :: if supported_icon_formats[icon_file_ext] and
+    it('finds icons with specified supported extension in search path', function()
+
+        -- Shimmed icon base directories contain the following icons:
+        --
+        --     .icons/icon5.png
+        --     .icons/icon6.xpm
+        --     .icons/icon7.svg
+
+        assert_found_in_path('icon5.png', '/.icons/icon5.png')
+        assert_found_in_path('icon6.xpm', '/.icons/icon6.xpm')
+        assert_found_in_path('icon7.svg', '/.icons/icon7.svg')
```

Comment:
this trick not gonna work :-)
 please get back to the original PR and really fix them

Reply:
What trick?

Reply:
it come up in my notifications so i did't noticed you're playing in your own repo :-)

Reply:
guess you just need to go to travis web site and put check down near repo name in the settings to enable travis for it

Reply:
It gave some mumbo-jumbo about a legacy version back then

I re-did it now and it may be in

I moved root as per your suggestion - fingers crossed

Reply:
Also wrong